charts/prefect-server/values.yaml

```diff
@@ -65,14 +59,11 @@ server:
   # sets PREFECT_LOGGING_SERVER_LEVEL
   loggingLevel: WARNING
 
-  ## If you want to run the UI seperately from the server, you will need to set the UI-specific URLs below
   uiConfig:
     # -- set PREFECT_UI_ENABLED; enable the UI on the server
     enabled: true
-    # -- sets PREFECT_UI_API_URL
-    prefectUiApiUrl: ""
-    # -- sets PREFECT_UI_URL
-    prefectUiUrl: ""
+    # -- sets PREFECT_UI_API_URL; If you want to connect to the UI form somewhere external to the cluster (like via an ingress), you need to set this value to the ingress URL (e.g. http://app.internal.prefect.com/api).
```

Comment:
@zzstoatzz @desertaxle - let me know if my description makes sense here

Reply:
Makes sense to me!